hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/PipelineMetadata.java

```diff
@@ -167,9 +149,19 @@ public PipelineMetadata build() {
       Preconditions.checkNotNull(datanodes);
       Preconditions.checkNotNull(replicationType);
 
-      return new PipelineMetadata(pipelineId, status, leaderNode, datanodes,
```

Comment:
Why was this changed from using the constructor? Setting the class field directly is non standard.